### PR TITLE
incremented width in sizedBox to add some more padding.

### PR DESCRIPTION
**Pull request by RishiMaersk**
Issue:- add more right padding to the 'run' button #2840
link:- https://github.com/dart-lang/dart-pad/issues/2840

Added more padding to the run button in dart pad.

### CODE REVIEW

File: pkgs/sketch_pad/lib/main.dart [hunk 1]
```diff
@@ -143,7 +143,7 @@ class _DartPadAppState extends State<DartPadApp> {
             ColorScheme.fromSeed(seedColor: lightPrimaryColor).copyWith(
           surface: lightSurfaceColor,
           onSurface: Colors.black,
-          surfaceVariant: lightSurfaceVariantColor,
+          surfaceContainerHighest: Colors.blue,
```
Comment (devoncarew) [inline pkgs/sketch_pad/lib/main.dart:146]: Why this change (and the similar one below)?

File: pkgs/sketch_pad/lib/widgets.dart [hunk 2]
```diff
@@ -124,7 +124,7 @@ class RunButton extends StatelessWidget {
               size: 20.0,
             ),
             SizedBox(
-              width: 8.0,
+              width: 16.0,
```
Comment (devoncarew) [inline pkgs/sketch_pad/lib/widgets.dart:127]: 👍

File: pkgs/sketch_pad/lib/main.dart (hunk 1 above)
Comment (RishiMaersk) [reply pkgs/sketch_pad/lib/main.dart:146]: I was getting an error in pipeline it was showing an error , which was exactly like this :- 

 info • 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre • lib/main.dart:146:11 • deprecated_member_use
   info • 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre • lib/main.dart:167:11 • deprecated_member_use
   info • 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre • lib/problems.dart:46:30 •

Comment (RishiMaersk) [reply pkgs/sketch_pad/lib/main.dart:146]: I thought may be changing it would resolve the error

Comment (devoncarew) [reply pkgs/sketch_pad/lib/main.dart:146]: It looks like we hit this in another PR as well: https://github.com/dart-lang/dart-pad/pull/2870/files#diff-ed60136a3d9ee7c71a9ed356332807808330f0cb99e3959050cfe26b087d6f8eR146. You could wait until that lands and then update your branch w/ the contents of `main`.

File: pkgs/sketch_pad/lib/widgets.dart (hunk 2 above)
Comment (RishiMaersk) [reply pkgs/sketch_pad/lib/widgets.dart:127]: https://github.com/dart-lang/dart-pad/pull/2870/files#diff-ed60136a3d9ee7c71a9ed356332807808330f0cb99e3959050cfe26b087d6f8eR146 . I believe this PR has been merged now and I have resolved the merge conflicts wrt main.

Comment (RishiMaersk) [reply pkgs/sketch_pad/lib/widgets.dart:127]: that surfaceVariant check is again failing

File: pkgs/sketch_pad/lib/main.dart [hunk 3]
```diff
@@ -143,8 +143,6 @@ class _DartPadAppState extends State<DartPadApp> {
             ColorScheme.fromSeed(seedColor: lightPrimaryColor).copyWith(
           surface: lightSurfaceColor,
           onSurface: Colors.black,
-          // TODO: Migrate to expanded Material 3 color palette.
-          // ignore: deprecated_member_use
```
Comment (devoncarew) [inline pkgs/sketch_pad/lib/main.dart:147]: We need these ignores for now to keep the CI green -

File: pkgs/sketch_pad/lib/widgets.dart (hunk 2 above)
Comment (RishiMaersk) [reply pkgs/sketch_pad/lib/widgets.dart:127]: so do I need to wait , before this depreciation issue is fixed with some other PR?

File: pkgs/sketch_pad/lib/main.dart (hunk 3 above)
Comment (RishiMaersk) [reply pkgs/sketch_pad/lib/main.dart:147]: do I need to wait , until this ignore part is resolved by some other PR ?

**Review by parlough — APPROVED** [review]
Thanks so much for this fix and your patience on getting this landed. Looks good to me!

I added back the ignore comments for now as accounting for the new Material 3 palette changes might require some larger considerations and changes.

**Comment by RishiMaersk** [comment]
Thanks for giving me the opportunity to work , this happens to be my first open source contribution
